journalism_workflow/static/journalism_workflow/copy_editor/js/controllers.js

```diff
@@ -0,0 +1,30 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('journalism_workflow.copy_editor.controllers')
+    .controller('CopyEditorController', CopyEditorController)
+
+  CopyEditorController.$inject = ['$scope', 'orchestraService'];
+
+  function CopyEditorController($scope, orchestraService) {
+    var vm = $scope;
+
+    // Store the article text document URL for easier summary later
+    var documentCreationStep = orchestraService.taskUtils.findPrerequisite(
+      vm.taskAssignment, 'document_creation')
+    vm.taskAssignment.task.data.articleDocument = documentCreationStep.task.data.articleURL;
+
+    // Set up the photos for captioning
+    vm.taskAssignment.task.data.photos = [];
+    var photoAdjustStep = orchestraService.taskUtils.findPrerequisite(
+      vm.taskAssignment, 'photo_adjust');
```

Comment:
PIL is pretty powerful, so if there's something specific you think could be done to make things more realistic, it's worth trying out. I'm still not sure I totally understand the line we're walking between 'toy example' and 'proof that orchestra can do powerful workflows'.

The reason I didn't go with resizing/thumbnails is that it would be difficult to see what changed without creating many copies as you describe. That's doable, but would be time-consuming. 

I'm inclined (only half out of laziness) to leave things as is. Perhaps we can create an issue to enhance the journalism workflow over time?